src/jdk.jlink/share/classes/jdk/tools/jmod/resources/jmod.properties

```diff
@@ -117,6 +117,7 @@ err.no.moduleToHash=No hashes recorded: no module matching {0} found to record h
 err.invalid.date=--date {0} is not a valid ISO-8601 extended offset date-time with optional time-zone format: {1}
 err.date.out.of.range=--date {0} is out of the valid range 1980-01-01T00:00:02Z to 2099-12-31T23:59:59Z
 err.compression.level.out.of.range=--compression-level {0} is out of the valid range: 0-9
+err.compression.level.wrong.mode=--compression-level {0} is only accepted with create mode
```

Comment:
Hello Aleksey, the place where this error message gets used doesn't right now pass the compression-level value to the constructor of `CommandException`. So I guess this usage of `{0}` here is unintentional?

Reply:
Yes, this should be fixed in new commits.